templates/redis.template.yml

```diff
@@ -86,4 +86,8 @@ hooks:
     - replace:
        filename: /etc/service/unicorn/run
        from: "# redis"
-       to: sv start redis || exit 1
+       to: |
+         if [ ! -d /shared/redis_data ]; then
+           install -d -m 0755 -o redis -g redis /shared/redis_data
+         fi
```

Comment:
@featheredtoast Is there a reason we can't just remove `tag: db` from 

https://github.com/discourse/discourse_docker/blob/6f8c17dbf4def3c991626b51eb9e33ccd92b2296/templates/redis.template.yml#L35-L38

Creating a folder even when we don't need it doesn't seem that bad. I think we should keep the runit service file as simple as possible.

Reply:
With the proposed build/configure/migrate separation, we'd be able to push built images that are theoretically able to just stand up with `docker run`, with no additional pups runs or launcher bootstraps, and no preconfigured environment.

It's that case where we'll need to ensure a redis data folder exists on boot.